src/integrator_interface.jl

```diff
@@ -415,6 +416,8 @@ function Base.iterate(tup::IntegratorTuples, state=0)
   return (tup.integrator.u,tup.integrator.t),state
 end
 
+Base.IteratorSize(::Type{IntegratorTuples{T}}) where T<:DEIntegrator = Base.SizeUnknown()
```

Comment:
Same here:
```suggestion
Base.IteratorSize(::Type{<:IntegratorTuples}) = Base.SizeUnknown()
```
It seems `eltype` is not defined here?